data/tutorials/lg_04_labels.md

```diff
@@ -256,8 +267,8 @@ to the next section.
 
 ### `Warning: This optional argument cannot be erased`
 We've just touched upon labels and optional arguments, but even this
-brief explanation should have raised several questions. The first may be
-why the extra `unit` argument to `open_window`? Let's try defining this
+brief explanation might have raised several questions. The first may be
+why use the extra `unit` argument to `open_window`. Let's try defining this
```

Comment:
I had to re-read this sentence a few times, could we quote the question:

The first may be:
> Why use the extra `unit` argument to `open_window`?

Let's try defining this function without the extra `unit`:

Reply:
I like that!